docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -61,7 +61,8 @@ image::images/alert-page-dropdown-controls.png[Alerts page with drop-down contro
 ====
 * You can have a maximum of four controls on the Alerts page.
 * You can't remove the *Status* control.
-* If you make any changes to the controls, you _must_ save the pending changes for them to persist.
+* If you make any changes to the controls, you _must_ save the pending changes for them to persist. 
+* Saved changes are stored in your browser's local storage, not your {ref}user-profile.html[user profile]. If you clear your browser's storage or log into your user profile from a different browser, the default controls on the Alerts page are reapplied.
```

Comment:
> If you clear your browser's storage or log into your user profile from a different browser, the default controls on the Alerts page are reapplied.

Somehow word `reapplied` seems not correct. How about below?

If you clear your browser's storage or log into your user profile from a different browser, you will lose your customizations and see default Controls.

Reply:
I think we can just go with "you'll lose your customizations". The part about your controls reverting to their default state is implied.

Reply:
Works for me. Thank you.

Reply:
```suggestion
* Saved changes are stored in your browser's local storage, not your {ref}user-profile.html[user profile]. If you clear your browser's storage or log into your user profile from a different browser, you will lose your customizations.
```